certif/tests/integration/components/new-certification-candidate-modal_test.js

```diff
@@ -86,6 +86,35 @@ module('Integration | Component | new-certification-candidate-modal', function (
     assert.dom(screen.getByLabelText('Certif complémentaire 2')).exists();
   });
 
+  test('[a11y] it should have some inputs required', async function (assert) {
```

Comment:
L'attribut required n'est pas directement lié à l'accessibilité, si ?

Si c'est une considération générale, je propose cette modification:
```suggestion
  test('it should have some inputs required', async function (assert) {
```

Reply:
Initialement j'avais indiqué "[a11y]" car je voulais mentionner également la présence de la mention "les champs marqués de * sont obligatoires" qui fait partie d'un besoin propre à l'a11y. Finalement comme je ne l'ai pas indiqué, je le supprime.